README.md

```diff
@@ -62,7 +62,7 @@ In [IBM Cloud](https://console.ng.bluemix.net/):
 
 ## 3. Building and Running
 
-> Note: This has been compiled and tested using Unity 2017.4.1f3 and Watson Unity SDK from the Unity asset Store as of May 2, 2018 and tested with the `develop` branch of the github unity-sdk as of `commit 9b439ca57bdb May2`.
+> Note: This has been compiled and tested using Unity 2018.2.of2 and Watson Unity SDK from the Unity asset Store as of July 24, 2018 and tested with the `develop` branch of the github unity-sdk as of `commit 44d8df33922 July 12`.
```

Comment:
2018.2.0f2? `0` vs `o`

Additionally you may want to add a note here about using Unity 2018.2 in regions other than US South because of the TLS 1.2 limitation. US South is the only region that has a TLS 1.0 enabled endpoint available for streaming and can be used with earlier version of Unity. Unity 2018.2 is the only version that supports TLS 1.2.

Reply:
OK, I've added a note. I didn't go into too much detail, as I'd prefer they just udate Unity.